src/app/pages/user/dialogs/update/update-user.dialog.ts

```diff
@@ -29,9 +46,14 @@ export class UpdateUserDialogComponent {
   }
 
   onUserTypeChange(event): void {
-    console.log('class value is: ' + this.data._class);
-    if (event.value === 'MANAGER') {
+    this.currentRole = event;
+
+    if (event === 'MANAGER') {
```

Comment:
can this "magic string" be avoided somehow? `MANAGER` is out of the enum values, right? so maybe something like `enum[event] === enum.MANAGER`?

Reply:
Yes sure, @dimkinv.
Will do on the dedicated refact PR,
since the fix will take care on Add user form, too.
Will that be fine ?

Reply:
Sure. Np